app/app/settings.py

```diff
@@ -76,8 +80,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql',
+        'HOST': os.environ.get('DB_HOST'),
+        'NAME': os.environ.get('DB_NAME'),
+        'USER': os.environ.get('DB_USER'),
+        'PASSWORD': os.environ.get('DB_PASS'),
```

Comment:
Nice!

For real-world scenarios we use https://django-configurations.readthedocs.io/en/stable/ (backend) and https://django-environ.readthedocs.io/en/latest/ (tk-hotels) which provide some extra goodies.